### PR TITLE
Eo/add child teams

### PR DESCRIPTION
**Pull request by eochoa260**

### CODE REVIEW

File: group.go [hunk 1]
```diff
@@ -16,6 +16,7 @@ type Group struct {
 	DescendantServicesConnection     Connection `json:"-" graphql:"descendantServices"`
 	DescendantSubgroupsConnection    Connection `json:"-" graphql:"descendantSubgroups"`
 	DescendantTeamsConnection        Connection `json:"-" graphql:"descendantTeams"`
+	ChildTeamsConnection             Connection `json:"-" graphql:"childTeams"`
```
Comment (rocktavious) [inline group.go:19]: With the way we wrote our sub queries - we no longer need this.  This is not used anywhere in the code it seems.  I suspect the other connection fields are also no longer used?

Comment (eochoa260) [reply group.go:19]: Yeah, since it gets the page info from the query within the ChildTeams function?

Comment (eochoa260) [reply group.go]: Removed all the *Connection fields

**Review by rocktavious — APPROVED** [review]
LGTM

**Pull request merged**